cluster/ci/monitoring/Makefile

```diff
@@ -11,13 +11,22 @@ prow-monitoring-deploy:
 	oc delete secret -n prow-monitoring alertmanager-prow --ignore-not-found=true
 	oc create -n prow-monitoring secret generic alertmanager-prow --from-file=./alertmanager.yaml --from-file=./msg.tmpl
 	oc apply -f ./alert_manager_expose.yaml
+	#grafana
+	oc create -n prow-monitoring secret generic grafana-config --from-file=./grafana.ini --dry-run -o yaml | oc apply -f -
+	oc create -n prow-monitoring configmap grafana-datasources --from-file=./datasources.yaml --dry-run -o yaml | oc apply -f -
```

Comment:
These are nice, but I think we also need to get auto-upload set up for them using `config_updater` -- add entries to [`plugins.yaml`](https://github.com/openshift/release/blob/4962d8240fbd20e43ae01af392ec2c8167257918/cluster/ci/config/prow/plugins.yaml#L702) for these, and think about using a configmap like `grafana-dashboards` with many entries if possible, so the config in `plugins.yaml` could just use a glob and we would not need to add new entries there when we add new dashboards.

Reply:
Also, due to a weirdness with how `config_updater` works, we will need the PR changing `plugins.yaml` to land first, before this one does.

Reply:
https://github.com/openshift/release/pull/3554